### DISCUSSION TITLE
Lack of actor property on Virtual Machine Activity

### DISCUSSION THREAD

**Discussion by overly-engineered** [ocsf/ocsf-schema#386] [Q&A] [answered]
I've been looking at the various events provided by OCSF, and many of them combine two or more entities into a single event. The virtual machine activity doesn't.

There is no actor field on it, so I can describe what happened to a given VM but not what the instigator of that change was.

What was the reason behind this decision?

**Comment by rroupski** [comment, marked as answer]
No reason -- we just missed it. Thank you!

I'll add the `Host, which add the `device` (where the virtual machine runs on) and `actor.user`.

**Reply by overly-engineered** [reply]
Awesome. Thank you!
